common/focuses/wc_magic_lifestyle_focuses.txt

```diff
@@ -0,0 +1,927 @@
+﻿light_magic_focus_1 = {
```

Comment:
if it's possible, move the icons to the right.
They overlap the lifestyle name.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/113175103-20362d00-925c-11eb-8473-a692c8bf49bd.png)

</details>

Reply:
Should be enough, I believe

Reply:
> 
> 
> Should be fine after [18235af](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/commit/18235afa805f777e80ef6471e7212158ae6b4219), I believe

Shadow, arcane, and nature overlap the name with the arrow.
![bild](https://user-images.githubusercontent.com/4825809/114279405-cfc98700-9a34-11eb-9b72-ab373ed3cf3f.png)

Reply:
@robmart, can you check it again?

Reply:
This is definitely solved

> 
> 
> @robmart, can you check it again?